packages/core/src/compiler/Style.test.ts

```diff
@@ -300,6 +300,113 @@ describe("Compiler", () => {
 
   // TODO: There are no tests directly for the substitution application part of the compiler, though I guess you could walk the AST (making the substitution-application code more generic to do so) and check that there are no Style variables anywhere? Except for, I guess, namespace names?
 
+  describe("Correct style programs with predicate aliasing", () => {
+    const domainProg = "type Set\npredicate IsSubset : Set s1 * Set s2";
+    const subProg = "Set A\nSet B\nSet C\nIsSubset(B,A)\nIsSubset(C,B)";
+    // TODO: Name these programs
+    const styProgs = [
+      // These are mostly to test setting shape properties as vectors or accesspaths
+      `Set x; Set y where IsSubset(x,y) as foo {}
```

Comment:
Use multi-line string with the ` ``` ` syntax for better readability.

Reply:
I'm not sure why, but forming strings with ````` causes typescript to complain (?)
<img width="706" alt="Screen Shot 2021-07-27 at 2 38 08 PM" src="https://user-images.githubusercontent.com/59301744/127209426-e1c9c095-1018-40d1-ac29-6a867778074e.png">

I think it may be because the style program strings are later concatenated with the canvas preamble, and it seems like in general typescript doesn't like concatenating ````` syntax strings? o_o
<img width="645" alt="Screen Shot 2021-07-27 at 2 39 40 PM" src="https://user-images.githubusercontent.com/59301744/127209646-5394266c-7cd7-4084-801c-1e458b752563.png">